.travis.yml

```diff
@@ -1,15 +1,25 @@
-# Don't use any predefined language env
-# (see https://github.com/travis-ci/travis-ci/issues/4895#issuecomment-150703192)
+# We will use docker to set up out environment, so don't use any particular
+# language in Travis itself
 language: generic
 
+# Enable docker
 sudo: required
+services:
+    - docker
 
-services: docker
+# Don't fetch submodules recursively, just a shallow fetch
```

Comment:
I though leaving no empty lines between both yml keys would be clear enough. What do you suggest?